playkit/src/main/java/com/kaltura/playkit/drm/DeferredDrmSessionManager.java

```diff
@@ -123,6 +129,33 @@ public DrmSession acquireSession(Looper playbackLooper, @Nullable EventDispatche
         return drmSessionManager.acquireSession(playbackLooper, eventDispatcher, format);
     }
 
+    private DrmSessionManager getDRMSessionManager(DrmCallback drmCallback, boolean allowClearLead, boolean isForceWidevineL3Playback) {
+        DrmSessionManager drmSessionManager;
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2 && isForceWidevineL3Playback) {
+            drmSessionManager = new DefaultDrmSessionManager.Builder()
+                    //.setUuidAndExoMediaDrmProvider(MediaSupport.WIDEVINE_UUID, FrameworkMediaDrm.DEFAULT_PROVIDER) // Not using default provider
```

Comment:
the if condition for L3 has to call .setUuidAndExoMediaDrmProvider in the new format 
else it has to be call in the previous way

                    .setUuidAndExoMediaDrmProvider(MediaSupport.WIDEVINE_UUID, FrameworkMediaDrm.DEFAULT_PROVIDER) // Not using default provider

Reply:
Done